Maple2.File.IO/Nif/NifM2dArchive.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Maple2.File.IO.Nif;
+
+public class NifM2dArchive {
```

Comment:
I feel like it might make more sense to extend `M2dReader` here? But also this really doesn't have any Nif specific logic so what's the point?

Reply:
Ah wait I lied, this does need to track the prefix with the M2dReader too for llid calculations. I could maybe just make it extend it for the sole purpose of holding that extra prefix string

Reply:
I would not mind just storing this extra field on `M2dReader` if it avoids the extra class. Just make sure this information is not already available.